docs/spec.md

```diff
@@ -6126,13 +6202,21 @@ works for per-tensor quantization. Per-axis quantization is work in progress
 ([#1574](https://github.com/openxla/stablehlo/issues/1574)).
 
 ```python
+
+def dequantize(x):
+  return (x - zero_point(x)) * scale(x)
+
+def quantize(x, type):
+  rounded_result = round_nearest_even(x / scale(type))
+  return clamp(storage_min(type), rounded_result + zero_point(type), storage_max(type))
```

Comment:
Similar questions apply to the definition of quantized DotGeneralOp. They don't have to be addressed in this PR, but let's at least open a ticket to nail this down.

Reply:
You know, perhaps we could use the same `dequantize_op_quantize` approach for DotGeneralOp? This would avoid the need to think specifically about DotGeneralOp - we'll be able to leverage whatever solution we end up with for `dequantize` and `quantize`.

Reply:
Also, if we're able to use `dequantize_op_quantize` for these ops, then we wouldn't need to worry about per-axis quantization for the underlying data movement ops, right?

Reply:
Opened https://github.com/openxla/stablehlo/issues/1628 for exploring the topic. Resolving it for now.